src/Integration.Vsix.UnitTests/ErrorList/IssuesSnapshotFactoryTests.cs

```diff
@@ -77,7 +77,7 @@ public void HandleFileRenames_NoReferencesToToRenamedFile_NoChanges()
             testSubject.CurrentSnapshot.Should().Be(snapshot.Object);
             location.CurrentFilePath.Should().Be("some file1");
 
-            snapshot.Verify(x=> x.IncrementVersion(), Times.Never);
+            snapshot.Verify(x=> x.GetUpdatedSnapshot(), Times.Never);
```

Comment:
These tests highlight that there is a problem between GetUpdatedSnapshot() and CreateUpdatedSnapshot(string) that is used for renaming. But that's a problem for another day...

Reply:
What's the problem?

Reply:
Just the readability issue. We're testing that we call CreateUpdatedSnapshot and not GetUpdatedSnapshot -- it's clumsy and confusing, especially when you see them side-by-side in the same test.